packages/discord.js-utilities/src/lib/type-guards.ts

```diff
@@ -36,18 +37,29 @@ export function isGroupChannel(channel: Channel): channel is PartialGroupDMChann
 }
 
 /**
- * Checks whether a given channel is a {@link GuildChannel}
+ * Checks whether or not a channel comes from a guild.
```

Comment:
```suggestion
 * Checks if a channel comes from a guild.
```
I'd use `if` here instead of `whether`, because `whether` relies on the "or not" and writing that out many times in the documentation is both tedious and verbose.